### ISSUE TITLE
Member with the same signature is already declared

### ISSUE THREAD

**Issue by damianh** [christianhelle/refitter#58]
**Describe the bug**
The following command:

    refitter https://api.hubspot.com/api-catalog-public/v1/apis/webhooks/v3 --namespace HubSpot --output ./GeneratedCode.cs

Generates the following:

```csharp
    public interface IWebhooksAPI
    {

        [Get("/webhooks/v3/{appId}/settings")]
        Task<SettingsResponse> GetAll(int appId);

        [Put("/webhooks/v3/{appId}/settings")]
        Task<SettingsResponse> Configure(int appId, [Body] SettingsChangeRequest body);

        [Delete("/webhooks/v3/{appId}/settings")]
        Task Clear(int appId);

        [Get("/webhooks/v3/{appId}/subscriptions")]
        Task<SubscriptionListResponse> GetAll(int appId);

        [Post("/webhooks/v3/{appId}/subscriptions")]
        Task Create(int appId, [Body] SubscriptionCreateRequest body);

        [Post("/webhooks/v3/{appId}/subscriptions/batch/update")]
        Task<BatchResponseSubscriptionResponse> UpdateBatch(int appId, [Body] BatchInputSubscriptionBatchUpdateRequest body);

        [Get("/webhooks/v3/{appId}/subscriptions/{subscriptionId}")]
        Task<SubscriptionResponse> GetById(int subscriptionId, int appId);

        [Delete("/webhooks/v3/{appId}/subscriptions/{subscriptionId}")]
        Task Archive(int subscriptionId, int appId);

        [Patch("/webhooks/v3/{appId}/subscriptions/{subscriptionId}")]
        Task<SubscriptionResponse> Update(int subscriptionId, int appId, [Body] SubscriptionPatchRequest body);

    }
```

Here there are two `GetAll` methods defined that are clashing.

**Support Key**: `kedfgny`

**OpenAPI Specifications**
https://api.hubspot.com/api-catalog-public/v1/apis/webhooks/v3

Also happens with https://api.hubspot.com/api-catalog-public/v1/apis/crm/v3/properties

**Comment by christianhelle** [comment]
@all-contributors please add @damianh for bugs

**Comment by christianhelle** [comment]
This issue in resolved in [v0.5.28](https://www.nuget.org/packages/Refitter/0.5.28)

**Comment by damianh** [comment]
Cool, will check it out.
